### PR TITLE
Template to create a storage bucket, copy files and add an acl.

### PR DESCRIPTION
**Pull request by oussamahammami**
Template to deploy a Cloud Storage Bucket, copies a list of files to it and create an ACL rule.

### CODE REVIEW

**Comment by ocsig** [comment]
Thank you for your great contribution, please continue!
At the moment we would like to collect all code samples coming from outside the core Deployment Manager team under the `community` folder.

Would you be able to move your sample? At the same time I'm reviewing your solution.

File: examples/v2/storage-bucket/storagebucket.jinja [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{#
+  20180717 - ohammami <osm.hammami@gmail.com>
+  version 0.1
+#}
+
+{% set ID = env['name'] %}
+
+resources:
+- name: {{ ID }}-bucket
```
Comment (ocsig) [inline examples/v2/storage-bucket/storagebucket.jinja]: 1.
I would advise not to use the `name` env variable for the bucket name, I would use a property.
2.
I would also remove the "bucket" string from the template and add it to the properties as well. ( Going further, I would write a helper function to generate the name of the bucket.)

**Comment by ocsig** [comment]
Its an interesting dilemma, but it feels like your `StorageBucket` template is not a generic template, it is fairly specific to your use case. Can you please rename the folder to avoid this confusion?

It would be also a great value, if you would create a Python equivalent as well.

**Comment by ocsig** [comment]
Please update your PR so we can approve it.

**Comment by oussamahammami** [comment]
the PR is updated.

**Comment by ocsig** [comment]
I encourage to extend your solution with a Python version. Would be great help for the community.

**Pull request merged**